principles.md

```diff
@@ -3,49 +3,44 @@
 Define a unit of software delivery called a Standard Container.
 The goal of a Standard Container is to encapsulate a software component and all its dependencies in a format that is self-describing and portable, so that any compliant runtime can run it without extra dependencies, regardless of the underlying machine and the contents of the container.
 
-The specification for Standard Containers is straightforward.
-It mostly defines 1) a file format, 2) a set of standard operations, and 3) an execution environment.
+The specification for Standard Containers defines:
 
-A great analogy for this is the shipping container.
-Just like how Standard Containers are a fundamental unit of software delivery, shipping containers are a fundamental unit of physical delivery.
+1. configuration file formats
+2. a set of standard operations
+3. an execution environment.
+
+A great analogy for this is the physical shipping container used by the transportation industry.
+Shipping containers are a fundamental unit of delivery, they can be lifted, stacked, locked, loaded, unloaded and labelled.
+Irrespective of their contents, by standardizing the container itself it allowed for a consistent, more streamlined and efficient set of processes to be defined.
+For software Standard Containers offer similar functionality by being the fundamental, standardized, unit of delivery for a software package.
 
 ## 1. Standard operations
 
-Just like shipping containers, Standard Containers define a set of STANDARD OPERATIONS.
-Shipping containers can be lifted, stacked, locked, loaded, unloaded and labelled.
-Similarly, Standard Containers can be created, started, and stopped using standard container tools (what this spec is about); copied and snapshotted using standard filesystem tools; and downloaded and uploaded using standard network tools.
+Standard Containers define a set of STANDARD OPERATIONS.
+They can be created, started, and stopped using standard container tools; copied and snapshotted using standard filesystem tools; and downloaded and uploaded using standard network tools.
```

Comment:
On Wed, Oct 07, 2015 at 10:12:46AM -0700, Vish Kannan wrote:
> > +They can be created, started, and stopped using standard
> > container tools; copied and snapshotted using standard filesystem
> > tools; and downloaded and uploaded using standard network tools.
> 
> Can we add more clarity here? What is required for hosting these
> containers to come up with standard network tools? Will OCI define
> APIs for these purpose?

Can we punt on this for now?  I think transfering these files with
tar/Nginx/curl, or tar/netcat, or IPFS, etc., etc., should be fine.
But this has been a hot-button issue in the past, and I'd rather avoid
having discussion about that stall this rewording PR.